### DISCUSSION TITLE
Error when deleting sub-file on read me

### DISCUSSION THREAD

**Discussion by Rockyread** [community/community#64837] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I wrote an extra file to create a snake game on my profile but I don't know how to delete it and I got my token exposed, how do I remove it from the readme page, I used manual measures to go to that file and press delete path but it still doesn't work ,

**Comment by mehulrajdev** [comment, marked as answer]
If your token is exposed, please contact (https://support.github.com/), and ask them to remove it, explaining how and what sensitive data was disclosed.
